apps/aehttp/src/sc_ws_handler.erl

```diff
@@ -294,10 +294,9 @@ websocket_info(?ERROR_TO_CLIENT(Err), #handler{protocol = Protocol} = H) ->
     {reply, {text, jsx:encode(Resp)}, H};
 websocket_info(stop, #handler{} = H) ->
     {stop, H};
-websocket_info({aesc_fsm, FsmPid, Msg}, #handler{fsm_pid = FsmPid,
-                                                 enc_channel_id = ChannelId,
```

Comment:
This undefined value was previously used irrespective of the result of the `set_channel_id/2` call.